chainstate/src/detail/tokens/check_utils.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common::chain::ChainConfig;
+use utils::ensure;
+
+use crate::TokensError;
+
+use super::error::CheckTokensError;
+
+pub fn check_token_text_length(text: &[u8], max_len: usize) -> Result<(), CheckTokensError> {
```

Comment:
Instead of passing the max_len, pass in the chain config. This function is kinda useless because you can pass any number you want.

Reply:
This function improve readability of `check_nft_issuance_data`. As the parameter, I use `chain_config` constants. Have a look, please. Because before this, `check_nft_issuance_data` looked dirty and unclear due to many code lines that did the same for different metadata items. But if you will confirm that this is unnecessary, I remove it.

Reply:
I hope, it's better now